### PR TITLE
Buildings fragment

### PR DESCRIPTION
**Pull request by rekagiday**

### CODE REVIEW

File: app/src/main/res/drawable/roundcorner.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+    <solid android:color="#FFFFFF" />
+    <corners android:radius="4dp" />
+</shape>
```
Comment (adamgyulavari) [inline app/src/main/res/drawable/roundcorner.xml]: @rekagiday new line at the end of file

File: app/src/main/res/layout/building_list_item.xml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        android:layout_height="60dp"
+        android:adjustViewBounds="false"
+        app:srcCompat="@drawable/town"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintBottom_toBottomOf="parent"
+        android:layout_marginLeft="8dp"
+        app:layout_constraintLeft_toLeftOf="parent"
+        android:layout_marginTop="8dp"
+        android:layout_marginBottom="8dp" />
+
```
Comment (adamgyulavari) [inline app/src/main/res/layout/building_list_item.xml:23]: @rekagiday unnecessary empty line

File: app/src/main/res/layout/fragment_buildings.xml [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+            android:divider="@android:color/transparent"
+            android:dividerHeight="16dp"
+            app:layout_constraintHorizontal_bias="0.0"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintRight_toRightOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/goldAndFoodBox" />
+
+    </android.support.constraint.ConstraintLayout>
+
+</FrameLayout>
```
Comment (adamgyulavari) [inline app/src/main/res/layout/fragment_buildings.xml]: @rekagiday new line

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/fragment_buildings.xml (hunk 3 above)
Comment (rekagiday) [reply app/src/main/res/layout/fragment_buildings.xml]: done

**Review by adamgyulavari — APPROVED** [review]

**Pull request merged**